index.html

```diff
@@ -594,52 +596,57 @@
 S_\mathrm{subsurface} = \mathrm{Slab}(f_\mathrm{dielectric}, V^\infty_\mathrm{subsurface}) \ .
 \end{equation}
 
-As for the [glossy-diffuse](index.html#model/dielectricbase/glossy-diffuse) slab and the [translucent-base](index.html#model/dielectricbase/translucentbase), the subsurface is bounded by a dielectric interface with BSDF $f_\mathrm{dielectric}$, which generates the primary specular lobe parametrized via the "specular" parameters as described in the Dielectric base section.
+As for the [glossy-diffuse](index.html#model/dielectricbase/glossy-diffuse) slab and the [translucent-base](index.html#model/dielectricbase/translucentbase), the subsurface is bounded by a dielectric interface with BSDF $f_\mathrm{dielectric}$, which generates the primary specular lobe parametrized via the "specular" parameters as described in the Dielectric base section. Superimposed on this is the reflection generated by scattering in the subsurface medium. In this case the subsurface medium $V^\infty_\mathrm{subsurface}$ is given a parametrization which is particularly convenient for controlling the volumetric effect of dense subsurface scattering:
 
-Superimposed on this is the reflection generated by scattering in the subsurface medium. In this case the subsurface medium $V^\infty_\mathrm{subsurface}$ is given a parametrization which is particularly convenient for controlling the volumetric effect of dense subsurface scattering:
+   - **`subsurface_radius`** * **`subsurface_radius_scale`**: the _mean free path_ (MFP) per RGB channel, $\mathbf{r}$, i.e. the typical distance that light travels in the medium before absorption or scattering. This thus controls the apparent density of the medium. In the limit of zero MFP, the medium tends towards infinite density, and approaches the look of an opaque diffuse surface. Being a length, **`subsurface_radius`** can be any value greater than or equal to zero. For convenience, we make the soft range $[0, 1]$, thus covering common cases where the MFP is typically much lower than the scene length units. The **`subsurface_radius_scale`** controls the color channel dependence of the MFP, and thus this color is visible in the light transmitted through thinner regions of the subsurface volume.
```

Comment:
> the typical distance that light travels in the medium before absorption or scattering.

"average" would be a little more precise than "typical" (though "typical" sounds fine if we're intentionally trying to sound more colloquial).

Also, it might potentially be useful to specify that this is the distance _along each individual ray_, maybe by saying something about a ray or a straight line.

Here's a possible alternative wording:

"the average distance that a ray of light travels through the medium before being absorbed or scattered"

Reply:
> we make the soft range $[0, 1]$, thus covering common cases where the MFP is typically much lower than the scene length units.

Makes sense, although I'm not sure about the assumption that it's typical and common for the MFP to be much lower than the scene length units. It really depends on what those units are and what's being rendered.

Reply:
> The **`subsurface_radius_scale`** controls the color channel dependence of the MFP, and thus this color is visible in the light transmitted through thinner regions of the subsurface volume.

This color will also be visible in shadowed regions even on an object without thin geometric features. For completeness, I would propose to mention both of these common cases where this color becomes visible. Here's one possible wording:

"and thus this color is visible in light transmitted through thinner features or into shadowed regions of the subsurface volume."

Reply:
>"the average distance that a ray of light travels through the medium before being absorbed or scattered"

Done in https://github.com/AcademySoftwareFoundation/OpenPBR/pull/171/commits/b1a88da5afcc24dbf5ce06aa3cca54679d223d2a

Reply:
>Makes sense, although I'm not sure about the assumption that it's typical and common for the MFP to be much lower than the scene length units. It really depends on what those units are and what's being rendered.

True we don't really know of course. Maybe we can soften the language and just say the MFP is just "typically lower" than the scene length units, not "much lower". I think that's probably true, e.g. most of the time SSS is used for characters being modelled in cm or m, and the MFP is of order mm.

Reply:
Makes sense. I think softening the language would be sufficient.

> thus covering common cases where the MFP is typically much lower than the scene length units

It's mostly the combination of "common", "typically", and "much" that makes me skeptical of the statement. "typically" seems redundant with "common", and "much lower" makes me question why the soft max distance even needs to be as high as 1. How about this instead:

"thus covering common cases where the MFP is lower than the scene length units"

Reply:
In https://github.com/AcademySoftwareFoundation/OpenPBR/pull/171/commits/3b0e2f1fddf6b017a893adb7b156a223e5e85ef3 altered it to:

>For convenience, we make the soft range [0,1], thus covering common cases such as skin where the MFP is lower than the scene length units.

I think it helps to give the informal example of "skin", as it seems intuitive that the MFP of skin is low.

Reply:
Sounds good.